packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AndroidPrompt/AndroidPrompt.tsx

```diff
@@ -27,21 +28,32 @@ export const AndroidPrompt = ({
   });
 
   const handleInstallClick = () => {
-    installPromptEvent.prompt();
+    try {
+      installPromptEvent.prompt();
 
-    // Wait for the user to respond to the prompt
-    installPromptEvent.userChoice.then((choiceResult) => {
-      if (choiceResult.outcome === 'accepted') {
-        // Hide after install prompt is accepted
-        console.log('User accepted the install prompt');
-        sessionStorage.setItem(HIDE_PROMPT, 'true');
-        setShowPrompt(false);
-      } else {
-        // Hide after install prompt is dismissed
-        sessionStorage.setItem(HIDE_PROMPT, 'true');
-        setShowPrompt(false);
+      // Wait for the user to respond to the prompt
+      installPromptEvent.userChoice.then((choiceResult) => {
+        if (choiceResult.outcome === 'accepted') {
+          // Hide after install prompt is accepted
+          console.log('User accepted the install prompt');
+          sessionStorage.setItem(HIDE_PROMPT, 'true');
+          setShowPrompt(false);
+        } else {
+          // Hide after install prompt is dismissed
+          sessionStorage.setItem(HIDE_PROMPT, 'true');
+          setShowPrompt(false);
+        }
+      });
+    } catch (e) {
+      console.error(e);
+      const manualStepsInstructionsEle =
```

Comment:
react state isn't used intentionally, since i didn't wanted to cause a re-render.